static/app/components/replays/header/replayMetaData.tsx

```diff
@@ -16,15 +21,32 @@ type Props = {
 };
 
 function ReplayMetaData({replayErrors, replayRecord}: Props) {
+  const location = useLocation();
+  const routes = useRoutes();
+  const referrer = getRouteStringFromRoutes(routes);
+  const eventView = EventView.fromLocation(location);
+
+  const DOMEventsTab = {
```

Comment:
nit: we'll usually have lower-case variable names, upper case reserved for react components.